drivers/dac/ad5770r/ad5770r.h

```diff
@@ -319,7 +323,10 @@ struct ad5770r_dev {
 struct ad5770r_init_param {
 	/* SPI */
 	spi_init_param				spi_init;
-
+	/* GPIO */
+	struct gpio_init_param			*nalarm;
+	struct gpio_init_param			*nreset;
```

Comment:
This is a matter of preferences, but we also have some precedence in this regard, I would name these with gpio_ prefix like `gpio_reset` or `gpio_reset_n` in this case, given it's active low.

Reply:
Will do this.